### PR TITLE
Create schema - PR

### PR DESCRIPTION
**Pull request by MichaelHassan**
This branch, creates tables , seeds them, creates connection and resetting scripts and updates the ReadME.

### CODE REVIEW

File: schema/connect.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source .env
```
Comment (Leon-JS02) [inline schema/connect.sh:1]: Handy bash scripts for connecting and resetting!

File: ERD.png
Comment (paulauus) [inline ERD.png:1]: Nice, good to be more specific with names and types.

File: schema/schema.sql [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    game_description TEXT ,
+    release_date TIMESTAMPTZ NOT NULL, 
+    is_NSFW BOOLEAN ,
+    image_URL TEXT , 
+PRIMARY KEY (game_id)
+);
+
+CREATE TABLE game_genre_assignment (
+    assignment_id INT GENERATED ALWAYS AS IDENTITY,
+    game_id INT NOT NULL UNIQUE ,
```
Comment (Leon-JS02) [inline schema/schema.sql:43]: I don't believe game_id and genre_id can be unique in these assignment tables. Multiple games can be in the table twice, as they have multiple genres.

File: terraform/README.md [hunk 3]
```diff
@@ -19,6 +19,8 @@ SUBNET_NAME="XXXX"
 VPC_ID="XXXX"
 ```
 
+Before running this terraform script, you need to go into the [web_scraping](../web_scraping), [report](../report),[load_to_rds](../load_to_rds) and [dashboard](../dashboard) folders and follow the instructions for `Deploying to the Cloud`, which involves making the ECRs containing the Docker images. Once the ECR's have been created then the terraform can be run.
```
Comment (Leon-JS02) [inline terraform/README.md:22]: I like how this links to different folders in the repo.

File: schema/schema.sql (hunk 2 above)
Comment (MichaelHassan) [reply schema/schema.sql:43]: nice spot

File: schema/README.md
Comment (mehrdadhalali) [inline schema/README.md:1]: Nice README structure

**Review by mehrdadhalali — APPROVED** [review]

**Pull request merged**